ci/windows_test.yml

```diff
@@ -0,0 +1,32 @@
+parameters:
+  sparkVersionUnderBar: ''
+  scalaVersion: ''
+
+steps:
+  - task: PowerShell@2
+    displayName: Download Winutils.exe
+    inputs:
+      workingDirectory: $(Build.BinariesDirectory)
+      pwsh: true
+      targetType: inline
+      script: |
+        echo "Download Hadoop utils for Windows."
+        curl -k -L -o hadoop.zip https://github.com/steveloughran/winutils/releases/download/tag_2017-08-29-hadoop-2.8.1-native/hadoop-2.8.1.zip
+        unzip hadoop.zip
+        New-Item -ItemType Directory -Force -Path hadoop\bin
+        cp hadoop-2.8.1\winutils.exe hadoop\bin
+        cp hadoop-2.8.1\hadoop.dll hadoop\bin
+        cp hadoop-2.8.1\hadoop.dll c:\windows\system32
```

Comment:
oh, this is required?

Reply:
Except for the spark 3.1 build, it is not necessary. (2.4, 3.0 tests success without copying it to system32 dir)

For the spark 3.1 build, it seems that sbt test doesn't reference hadoop/bin/hadoop.dll so it makes the tests fail.
I don't fully understand the reason. Maybe Spark 3.1 doesn't reference HADOOP_HOME to include dynamic libraries.